src/components/input-collaborator.js

```diff
@@ -0,0 +1,342 @@
+/** @jsx jsx */
+import React from "react";
+import { jsx } from "@emotion/core";
+
+/*
+  function CreateChannel({ channels, onCreate }) {
```

Comment:
Hi, I think it would look better if you delete this comment

Reply:
Yes sir!